examples/config.yaml

```diff
@@ -13,6 +13,10 @@ auth:
     clientSecret: xxxxxxxxxxxxxxxxxxxxxxxxxxxx
     scopes:
       - profile
+  x509:
+    certPath: /etc/ssl/certs/floodgate.crt
+    enabled: true
+    keyPath: /etc/ssl/private/floodgate.key
```

Comment:
Will relative path work here? If so, maybe better path in example file is something like `~/.config/floodgate/floodgate.key` ?